src/test/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImplTest.java

```diff
@@ -267,4 +312,134 @@ private MediaRequestEntity createMediaRequest(OffsetDateTime startTime, OffsetDa
         mediaRequest.setEndTime(endTime);
         return mediaRequest;
     }
+
+    @Test
+    void testNotifyUserScheduleErrorNotification() {
+        List<DefendantEntity> defendantList = new ArrayList<>();
+        defendantList.add(mockDefendantEntity);
+        defendantList.add(mockDefendantEntity);
+
+        initNotifyUserScheduleErrorNotificationMocks(MOCK_HEARING_DATE, MOCK_COURTHOUSE_NAME, TIME_12_00, TIME_13_00);
+        when(mockDefendantEntity.getName()).thenReturn(MOCK_DEFENDANT_NAME);
+        when(mockCourtCase.getDefendantList()).thenReturn(defendantList);
+
+        audioTransformationService.notifyUser(mockMediaRequestEntity, mockCourtCase, NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString());
+        verify(mockNotificationApi).scheduleNotification(dbNotificationRequestCaptor.capture());
+        var actual = dbNotificationRequestCaptor.getValue();
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("HH:mm:ss");
+
+        assertEquals(actual.getTemplateValues().get(AUDIO_START_TIME), TIME_12_00.format(formatter));
```

Comment:
You've got these the wrong way around - it should be 'expected' first followed by 'actual'.

Looks like you've got the same issue all over the place, so please update accordingly.

Reply:
Yes, Spotted that this morning when I saw the code coverage issue.